migration-helper/serverless-to-raw.sh

```
@@ -0,0 +1,1497 @@
+    if [[ "$isvc_count" -eq 0 ]]; then
+        log_error "No InferenceServices found in namespace '$NAMESPACE'"
+        echo "There are no models to migrate."
+        exit 1
+    fi
+
+    # Get names of InferenceServices that are eligible (Serverless deployment mode)
+    local isvc_names=()
+    while IFS= read -r name; do
+        if [[ -n "$name" && "$name" != "null" ]]; then
+            # Check if this InferenceService has Serverless deployment mode
+            local deployment_mode=$(echo "$isvc_list" | yq ".items[] | select(.metadata.name == \"$name\") | .metadata.annotations.\"serving.kserve.io/deploymentMode\" // \"\"")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)